src/editors/containers/TextEditor/components/RawEditor/CodeEditor/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useRef } from 'react';
```

Comment:
Should we pull out CodeEditor from under RawEditor since both CodeEditor and SourceCodeModal accesses it?